dev-util/vulkan-headers/vulkan-headers-1.2.189.0.ebuild

```diff
@@ -10,9 +10,9 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="https://github.com/KhronosGroup/${MY_PN}.git"
 	inherit git-r3
 else
-	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/sdk-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
There's a `sdk-1.2.189.1` tag in Vulkan-Headers. Should we go for that instead of .0?

I'd prefer a version bump to .1 rather than changing a stable ebuild's distfile.